JS-113_javascript-objects/loop-through-objects/code-challenge/display-an-array-of-objects-on-the-page/js/pets.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * JS-112: JavaScript Arrays - Loop Through Arrays
+ * 
+ * Code challenge: Display An Array of Objects on the Page
+ */
+const pet = [];
+
+function createPet(name, type, breed, age, photo) {
+  return {
+    name: name,
+    type: type,
+    breed: breed,
+    age: age,
+    photo: photo
+  };
+}
+
+pet.push(createPet('Sussie', 'Dog', 'Australian Shepherd', 4, 'img/aussie.jpg'));
+pet.push(createPet('Hotdog', 'Dog', 'Dachshund', 5, 'img/dachshund.jpg'));
+pet.push(createPet('Jeff', 'Dog', 'Golden Retriever', 6, 'img/golden.jpg'));
+pet.push(createPet('Perry', 'Cat', 'Persian', 4, 'img/persian.jpg'));
+pet.push(createPet('Dude', 'Dog', 'Pug', 3, 'img/pug.jpg'));
+pet.push(createPet('Miming "the ulam thief"', 'Cat', 'Domestic Shorthair', 100, 'img/tabby.jpg'));
```

Comment:
See the comment above. Important things here are variable names should implicitly convey their purpose and the DRY (don't repeat yourself) principle.